packages/react-components/theme-designer/src/components/ColorTokens/ColorTokens.tsx

```diff
@@ -2,20 +2,15 @@
 import * as React from 'react';
 import { makeStyles } from '@griffel/react';
 import { Button, Caption1 } from '@fluentui/react-components';
-import { Brands, BrandVariants, teamsDarkTheme, teamsLightTheme, Theme } from '@fluentui/react-theme';
+import { Brands, teamsDarkTheme, teamsLightTheme, Theme } from '@fluentui/react-theme';
```

Comment:
_nit:_ Import from `@fluentui/react-components` and separate `import type` statements

```suggestion
import { makeStyles, teamsDarkTheme, teamsLightTheme, Button, Caption1 } from '@fluentui/react-components';
import type { Brands, Theme } from '@fluentui/react-components';
```